src/Components/GoodsList/ListItem.tsx

```diff
@@ -0,0 +1,5 @@
+import { ListItemProps } from '../../types/list/ListItemProps';
+
+export const ListItem: React.FC<ListItemProps> = ({ good }) => (
+  <li data-cy="Good">{good}</li>
```

Comment:
According to the checklist item 2, when rendering a list, you should add a `key` prop to each list item. In this case, you should ensure that each <li> element has a unique `key` prop based on the good's identifier.